inc/c_util/constbuffer_array_tarray.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+#ifndef CONSTBUFFER_ARRAY_TARRAY_H
+#define CONSTBUFFER_ARRAY_TARRAY_H
+
+#ifdef __cplusplus
+#include <cstdint>
+#else
+#include <stdint.h>
+#endif
+
+#include "c_util/tarray.h"
+#include "c_pal/thandle.h"
```

Comment:
Can we have c_pal before c_util so that c_util includes are also grouped together?

Usually the order we use is:
- macro_utils
- c_pal
- c_util

etc.

Reply:
done